models/intercom__conversation_enhanced.sql

```diff
@@ -73,6 +73,8 @@ enriched as (
 
         latest_conversation.source_subject as conversation_subject,
         case when (latest_conversation.assignee_type is not null) then latest_conversation.assignee_type else 'unassigned' end as conversation_assignee_type,
+        case when (coalesce(lower(latest_conversation.assignee_type), '') = 'team') then latest_conversation.assignee_id else null end as team_assignee_id,
+        case when (coalesce(lower(latest_conversation.assignee_type), '') = 'admin') then latest_conversation.assignee_id else null end as admin_assignee_id,
```

Comment:
This is query is only looking to the latest assignee_type within the conversation. With this, if a team was assigned at the start, and then other assignees have been designated, then there will be no record of the team assignment.

I believe we will need to do a bit more to accurately capture team assignment in this output model. Similar to how we already capture first/last assignment for admin we will want to do that for teams. We can chat more about this during standup.

Reply:
Additionally, we have test intercom data that I can send over to you to help seed and troubleshoot.